wakuv2/waku.go

```diff
@@ -126,7 +127,7 @@ type Waku struct {
 	storeMsgIDs   map[gethcommon.Hash]bool // Map of the currently processing ids
 	storeMsgIDsMu sync.RWMutex
 
-	connStatusChan          chan node.ConnStatus
+	topicHealthStatusChan   chan peermanager.TopicHealthStatus
```

Comment:
For sure the usage of the topicHealtStatus channel needs to be improved. I only added the minimum needed code so the existing functionality is maintained since the connection status feature was removed from go-waku as it did not make sense in the context of having multiple pubsub topics.

Proper usage of this channel must be implemented, following recommendations specified in https://github.com/status-im/status-go/issues/4628